dist/image-sequencer.js

```diff
@@ -47742,12 +47761,13 @@ ImageSequencer = function ImageSequencer(options) {
     setUI: setUI,
     exportBin: exportBin,
     modulesInfo: modulesInfo,
+    toString: toString,
     
     //other functions
     log: log,
     objTypeOf: objTypeOf,
     copy: copy,
-
+    
```

Comment:
watch for extra spaces sneaking in!

Reply:
@jywarren How do I prevent these?? like if I autoIndent these spaces will appear according to the indentation of the project right??
So should I not AutoIndent??
I am currently using VScode

Reply:
Hm, it's not a big deal, but I can manually delete inserted indents when I see them, i use `vim`. Indentation config is always a bit tough. Just do your best!

Reply:
Oh I'll research about this a little! Right now I'll do it manually, Thanks :+1: